lib/Backend/Inline.cpp

```diff
@@ -2385,12 +2425,12 @@ IR::Instr * Inline::InlineApplyWithArgumentsObject(IR::Instr * callInstr, IR::In
     InsertInlineeBuiltInStartEndTags(callInstr, 3, &builtInStartInstr); //3 args (implicit this + explicit this + arguments = 3)
 
     // Move argouts close to call
-    IR::Instr* argInsertInstr = builtInStartInstr;
+    /*IR::Instr* argInsertInstr = builtInStartInstr;
     builtInStartInstr->IterateArgInstrs([&](IR::Instr* argInstr) {
         argInstr->Move(argInsertInstr);
         argInsertInstr = argInstr;
         return false;
-    });
+    });*/
```

Comment:
yes, thanks

---
In reply to: [85820369](https://github.com/Microsoft/ChakraCore/pull/1838#discussion_r85820369) [](ancestors = 85820369)